### PR TITLE
cmus recipe (x86)

### PR DESCRIPTION
**Pull request by miqlas**
It can't find opus and libcue, but works with mp3, etc...

### CODE REVIEW

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	cmd:make
+	cmd:m4
+	cmd:pkg_config${secondaryArchSuffix}
+	cmd:sed
+	cmd:find
+	cmd:g++$secondaryArchSuffix
+	cmd:cmake
+	"
+
+PATCHES="cmus-2.7.1-gcc2.patch"
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: This goes right under CHECKSUM.

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 2]
```diff
@@ -0,0 +1,91 @@
+SUMMARY="Curses based Music Player"
+DESCRIPTION="
+cmus is a small, fast and powerful console music player for Unix-like operating systems.\
+"
+HOMEPAGE="https://cmus.github.io/"
+SOURCE_URI="https://github.com/cmus/cmus/archive/v2.7.1.tar.gz"
+CHECKSUM_SHA256="8179a7a843d257ddb585f4c65599844bc0e516fe85e97f6f87a7ceade4eb5165"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2005-2015 The cmus Project"
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: Swap the order of this and LICENSE and put it under HOMEPAGE.

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 3]
```diff
@@ -0,0 +1,91 @@
+SUMMARY="Curses based Music Player"
+DESCRIPTION="
+cmus is a small, fast and powerful console music player for Unix-like operating systems.\
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: Get rid of trailing `\` and move the `"` to this line.

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+PATCHES="cmus-2.7.1-gcc2.patch"
+
+BUILD()
+{
+
+	#	ToDo: We need to set the default output plugin in cmus:
+	#	:set output_plugin=ao
+
+	./configure  \
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: Is this an Autotools configure? If not, add a comment saying `not an autotools configure`.

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+	cmd:cmake
+	"
+
+PATCHES="cmus-2.7.1-gcc2.patch"
+
+BUILD()
+{
+
+	#	ToDo: We need to set the default output plugin in cmus:
+	#	:set output_plugin=ao
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: See if there's a way to create a GLOBAL_WRITEABLE_FILES for this.

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+		LDFLAGS=-lnetwork
+
+	make $jobArgs
+
+}
+
+INSTALL()
+{
+
+	make install
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: Get rid of the superflous blank lines, here and in BUILD.

**Comment by waddlesplash** [comment]
Also, the commit is not linked to your GitHub account -- please change the email in the commit, or add that email to your account so they are linked.

File: media-sound/cmus/cmus-2.7.1.recipe (hunk 5 above)
Comment (miqlas) [inline media-sound/cmus/cmus-2.7.1.recipe]: Based on my tests it picks it up automatically, so no change needed.

**Comment by miqlas** [comment]
waddlesplash: i changed everything, i hope it is ok now :)

**Comment by waddlesplash** [comment]
Except now you have 4 commits and one is a merge. Please have only one commit. (You might need to start from a fresh Git branch to get this.)

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 7]
```diff
@@ -0,0 +1,84 @@
+SUMMARY="A Console Based Audio Player "
+DESCRIPTION="
+cmus (C* MUsic Player) is a small, fast and powerful console audio player which supports most major audio formats. Various features include gapless playback, ReplayGain support, MP3 and Ogg streaming, live filtering, instant startup, customizable key-bindings, and vi-style default key-bindings."
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: This should be wrapped to 80 characters using \s, you made it longer than it was before...

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 8]
```diff
@@ -0,0 +1,84 @@
+SUMMARY="A Console Based Audio Player "
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: Use sentence casing, get rid of ending ` `.

File: media-sound/cmus/cmus-2.7.1.recipe [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+INSTALL()
+{
+	mkdir -p $prefix
+	mkdir -p $binDir
+	mkdir -p $dataDir
+	mkdir -p $libDir
+	mkdir -p $manDir
+	mkdir -p $docDir
```
Comment (waddlesplash) [inline media-sound/cmus/cmus-2.7.1.recipe]: Is this needed? Usually, `make install` detects/creates those dirs as needed.

**Comment by miqlas** [comment]
FFFFUUUUU...
Need to redone it again...

**Comment by waddlesplash** [comment]
No, you can use `git push --force` to overwrite the remote branch.

**Pull request closed without merging**